src/Utilities/hooks/useBulkSelectConfig.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable camelcase */
+import { useDispatch, useStore } from 'react-redux';
+import * as actions from '../../store/actions';
+import React, { useCallback, useRef } from 'react';
+import useFetchBatched from './useFetchBatched';
+import { flatten, map } from 'lodash';
+import { loadSystems } from '../sharedFunctions';
+import { inventoryCache } from '../../components/InventoryTable/InventoryTable';
```

Comment:
Can we not use inventoryCache? It can create bugs here as it is prone have an older filter.